src/styles/components/badges/styles.less

```diff
@@ -16,163 +16,167 @@ components/badges.less
 /* -----------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 
-Badge
+Badges
 
 --------------------------------------------------------------------------------
 ----------------------------------------------------------------------------- */
 
-.badge {
+& when (@badges-enabled) {
 
-  color: @body-background-color;
-  font-size: @body-text-font-size;
-  line-height: @body-text-line-height;
-  padding: 0 @base-spacing-unit * 0.25;
```

Comment:
http://www.slate.com/articles/health_and_science/science/2013/03/facebook_math_problem_why_pemdas_doesn_t_always_give_a_clear_answer.single.html

tl;dr use parentheses to avoid any chance of confusion.